README.md

```diff
@@ -201,6 +201,18 @@ export function consumeToolBar(getToolBar) {
     callback: 'application:about'
   });
 
+  // Buttons can be styled with arbitrary CSS through classes
+  toolBar.addButton({
+    icon: 'octoface',
+    callback: 'application:about',
+    class: 'my-awesome-class'
```

Comment:
It would more useful if you provide a full example that includes the class definition.

Reply:
Good suggestion! I would like some feedback from @suda on how they would like to incorporate that into the documentation.

Some possible options:
- Add a new file for it `docs/button-with-class.md` (perhaps a bit overkill, though other options might benefit from a similar approach?)
- Add it to [the repo wiki](https://github.com/suda/tool-bar/wiki)
- Create a gist with an example and link there

Reply:
I think if the class definition is short it can be put at the end of the main readme. But if it takes a lot of space the separate markdown file is a better option. Gist works too, but markdown file stays in this repository.

I don't think repo Wiki is suitable for this case.

Reply:
> I think if the class definition is short it can be put at the end of the main readme. But if it takes a lot of space the separate markdown file is a better option.

I'm mainly just not sure where to put it... I have [a comparable thing in the README](https://github.com/ericcornelissen/pinned-tabs-for-atom#customization) for one of my own packages, but the use case is slightly different here...

> Gist works too, but markdown file stays in this repository.

Wanted to add that disclaimer, but thought it was self-evident. Thanks for pointed it out explicitly though!

> I don't think repo Wiki is suitable for this case.

Yea, I'm not sure about it either, but it's definitely a valid option :smile: 

In the end, again, it's up to @suda

Reply:
I think adding it to the readme is perfectly fine :) The [Example](https://github.com/suda/tool-bar#example) section covers all possible options and we can probably add it there :)

Reply:
I put something together, let me know what you think :smile: (and I rebased the PR)